### ISSUE TITLE
Heroku thinks strider v1.7.5 is Ruby app?

### ISSUE THREAD

**Issue by bencooling** [Strider-CD/strider#895]
Simple installation of Strider & initial push to heroku:
```
git clone https://github.com/Strider-CD/strider.git && cd strider
heroku create
git push heroku master
```

Heroku response:
```
...
remote: -----> Ruby app detected
...
```

**Comment by bencooling** [comment]
Just submitted a pull request with a fix:
https://github.com/Strider-CD/strider/pull/896

**Comment by knownasilya** [comment]
Even though it says Ruby, did the install work correctly?

**Comment by bencooling** [comment]
It seems to copy the repo files to heroku fine, but no node binary, no node_modules directory:

```
Running bash on young-savannah-4714... up, run.3880
~ $ node bin/strider addUser
bash: node: command not found
```

**Comment by michaelmior** [comment]
Did you try using the Heroku button? Alternatively, did you try setting `BUILDPACK_URL` after calling `heroku create`? I haven't tested this recently, but when I first added Heroku support, I used [heroku-buildpack-multi](https://github.com/ddollar/heroku-buildpack-multi) so we could have Node.js, Python, and Ruby support. While your PR does fix things for Node.js, it will cause issues with trying to test repos in Python and Ruby.
